ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -40,6 +40,7 @@ var (
 	log                       = logf.Log.WithName("raycluster-controller")
 	DefaultRequeueDuration    = 2 * time.Second
 	PrioritizeWorkersToDelete bool
+	ForcedClusterUpgrade      = true
```

Comment:
should it be `bool` like `PrioritizeWorkersToDelete` and accept values from `main.go`?